### PR TITLE
Add a few more companies

### PR DESCRIPTION
**Pull request by pradip7374**
- Add Dolphin Dive Technology Pvt. Ltd.
- Add Outsource Array
- Add Supreme IT Solutions Pvt. Ltd.

### CODE REVIEW

**Comment by mesaugat** [comment]
@pradip7374 Thank you for the PR. Can you `rebase` the latest changes in master?

**Comment by pradip7374** [comment]
Done @mesaugat

File: README.md [hunk 1]
```diff
@@ -129,6 +130,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [NepXpress Cargo & Courier](https://www.nepxpress.com) | Buddhanagar, Kathmandu | Instant Home Delivery Service in Nepal
 - [NITV Streamz Pvt. Ltd.](http://nettv.com.np/) | Ekantakuna, Lalitpur | TV streaming
 - [nLocate](https://nlocate.com) | Harihar Bhawan, Lalitpur | Locate services, businesses, events around you
+- [Outsource Array](https://outsourcearray.com/) | Thamel, Kathmandu | Design, Development, Maintenance and Support all in one place. Here at Outsource Array, we offer tech solutions that engage your customers and deliver outstanding results. Couple that with exceptional manual tech support and you will get a first class IT Services in all fronts
```
Comment (mesaugat) [inline README.md]: The description is too long here. Can we make it short?

File: README.md [hunk 2]
```diff
@@ -168,6 +171,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Three Monks](http://www.threemonk.com/) | New Baneshwor, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
 - [Truemark](https://www.truemark.com.np/) | Bagbazaar, Kathmandu | Software Development Company
 - [UBA Solutions](http://www.uba-solutions.com/) | Thadodhunga, Lalitpur | Software Research and Development
+- [Umanga Creation Pvt. Ltd.](https://umangacreation.com.np/) | Lazimpat-2 Uttar Dhoka, Kathmandu |We innovate/create/implement brands to realise their true potential
```
Comment (mesaugat) [inline README.md]: @pradip7374 Thank you for the pull request. I am requesting you a few changes.

Only `Lazimpat, Kathmandu` or `Uttar Dhoka, Kathmandu`. Let's remain consistent with other companies on the list.

Need a space after `|`. Remove `We`.

```diff
- We
```

Can you make the description like this? `Innovate and create brands to realize their true potential`

File: README.md [hunk 3]
```diff
@@ -154,6 +156,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Spark Tech](http://spark.com.np/) | Tinkune, Kathmandu | Tech agency that creates innovative and meaningful digital products
 - [SPI Nepal](http://www.softvision.com/) | Thapathali, Kathmandu | Provider of Digital Solutions, Mobility, Analytics, Merchandise Planning, In-Store Engagements and Quality Assurance
 - [SUBISU CABLENET (P.) LTD.](https://subisu.net.np/) | Jhamsikhel, Lalitpur | Internet, Network and Television Service Provider
+- [Supreme IT Solutions Pvt. Ltd.](https://supremeit.net/) | New Baneshwor, Kathmandu | What we do is important
```
Comment (mesaugat) [inline README.md]: This description doesn't say anything about the company. Can you please update it? @pradip7374

File: README.md [hunk 4]
```diff
@@ -65,6 +65,7 @@ You can sort/filter/download the data from [Airtable](https://airtable.com/shrXv
 - [Danfe Solution Pvt. Ltd.](http://danfesolution.com/) | Gyaneshwor, Kathmandu | IT Company focusing on Web Development, Mobile Application and Desktop Application
 - [Danphe Software Labs Pvt. Ltd.](http://www.danpheit.com/) | Anamnagar, Kathmandu  | We specialize in Software and Web Development
 - [Deerwalk](http://www.deerwalk.com/) | Sifal, Kathmandu | Bringing you software solutions for POPULATION HEALTH MANAGEMENT
+- [Dolphin Dive Technology Pvt. Ltd.](https://dolphindivetech.com/) | Jhamsikhel, Lalitpur |  US Based Outsourcing Agency
```
Comment (mesaugat) [inline README.md]: @pradip7374 Need a good description for the company here. `US Based Outsourcing Agency` doesn't say much. Also, let's remove the region (US) from the description and focus on what the company does best.

**Comment by pradip7374** [comment]
Please check now @mesaugat

**Pull request merged**